src/main/java/com/github/seikichi/pronto_checkstyle_findbugs_example/App.java

```diff
@@ -6,6 +6,9 @@ public String getGreeting() {
   }
 
   public static void main(String[] args) {
+    if ("foo" == String.format("%soo", "f")) { // findbugs violation
```

Comment:
findbugs Comparison of String objects using == or != in com.github.seikichi.pronto_checkstyle_findbugs_example.App.main(String[])